cassandra-diagnostics-core/src/test/java/io/smartcat/cassandra/diagnostics/module/slowquery/SlowQueryModuleTest.java

```diff
@@ -0,0 +1,45 @@
+package io.smartcat.cassandra.diagnostics.module.slowquery;
+
+import io.smartcat.cassandra.diagnostics.Measurement;
+import io.smartcat.cassandra.diagnostics.Query;
+import io.smartcat.cassandra.diagnostics.config.ConfigurationException;
+import io.smartcat.cassandra.diagnostics.module.ModuleConfiguration;
+import io.smartcat.cassandra.diagnostics.module.TestReporter;
+import io.smartcat.cassandra.diagnostics.reporter.Reporter;
+import io.smartcat.cassandra.diagnostics.reporter.ReporterConfiguration;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.internal.util.collections.Sets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Collections;
+
+public class SlowQueryModuleTest {
+
+    private SlowQueryModule module = null;
+
+    @Before
+    public void init() throws ConfigurationException {
```

Comment:
There is no need for init method since no preparation is done except initialization. Having a class variable module doesn't make sense since we will have to test with different types of reporters thus having to create a new module instance. We can keep module reference under test scope.